guide/english/certifications/javascript-algorithms-and-data-structures/basic-javascript/use-the-conditional-ternary-operator/index.md

```diff
@@ -20,7 +20,7 @@ Remember that the "traditional" `if...else` syntax can be re-written using the c
 
 ```javascript
 function checkEqual(a, b) {
-  return a === b ? true : false;
+  return a === b ? "Equal" : "Not Equal";
 }
 ```
 ·  Run code at [repl.it](https://repl.it/@AdrianSkar/Basic-JS-Ternary-operator).
```

Comment:
You'd also need to do something about this

Reply:
@thecodingaviator, want to confirm that changing the content in that repl.it link and updating the link with https://repl.it/@rupali317/Basic-JS-Ternary-operator is fine right?

Reply:
I made the change